sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/controller/container/EmptyThroughputContainerController.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.throughputControl.controller.container;
+
+import com.azure.cosmos.implementation.RxDocumentServiceRequest;
+import reactor.core.publisher.Mono;
+
+/**
+ * This is a dummy throughput container controller, which will just let the requests pass by.
+ * This will be created when there is no throughput control group defined for the container.
+ */
+public class EmptyThroughputContainerController implements IThroughputContainerController {
+    @Override
+    public boolean canHandleRequest(RxDocumentServiceRequest request) {
+        return true;
+    }
+
+    @Override
+    public Mono<Void> close() {
+        return Mono.empty();
```

Comment:
In case you choose to remove `Mono<Void>` from the interface, this can be a no-op.

Reply:
As discussed offline, will keep init and close as Mono now